src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -45,11 +50,17 @@
 import api.support.builders.FixedDueDateSchedulesBuilder;
 import api.support.builders.LoanBuilder;
 import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.NoticeDescriptorBuilder;
+import api.support.builders.NoticePolicyBuilder;
 import api.support.builders.UserBuilder;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
 public class CheckOutByBarcodeTests extends APITests {
+
+  private static final DateTimeFormatter NOTICE_DATE_TIME_FORMATTER =
+    DateTimeFormat.forPattern("MM/dd/yyyy hh:mm a");
```

Comment:
Where does this date format come from?

Reply:
There is no requirements on how due date should be displayed in patron notice. I've added this as stub.  Do you think we should something default like rfc3339 for now?

Reply:
If we are intending to present this in the notice then I think we need to clarify the requirements with Darcy, otherwise this is likely to be rejected during testing.

Please can you ask her what format she wants to use?

Why is this being decided here, rather than by the templating part of the system?

Reply:
I've asked about the format in UICIRC-154 and the answer is to use local date format.
Seems like now it's out of scope for this story.
And the decision where to apply the locale could be taken later.
Also I should mention that the template engine now does not support date time formatting (only strings can be used to fill placeholders in templates).   
What do you think, can we leave this as stub now?

Reply:
@dmytrotur Please can you raise a story for making this decision and designing a way to do it in a way that respects the chosen locale.

Personally, I would've preferred to use a less opinionated placeholder, that made it obvious it was such and could be acceptable irrespective of locale, e.g. RFC 3339 like you suggested above.

I also believe this really should be the responsibility of the templating system, rather than the system asking to send a templated notice.

I think we are out of time for the 2019 Q1 release to do this, as this work in the current form is intended to be included.

Reply:
@marcjohnson-kint Ok, I'll raise this story.

Reply:
Should I change date format to 'RFC 3339'? Or are we going to merge this as is?

Reply:
@dmytrotur Personally, if it is a small change, I think that would be good.

It makes it explicit that this is potentially temporary. And, at least in theory, could be acceptable across locale choices.

Feel free to check with @kostyantyn-kh if needed.

Reply:
@marcjohnson-kint I agree, I'll do this.